consensus/clique/clique.go

```diff
@@ -657,7 +659,7 @@ func (c *Clique) Seal(chain consensus.ChainHeaderReader, block *types.Block, res
 		}
 	}
 	// Sweet, the protocol permits us to sign the block, wait for our time
-	delay := time.Unix(int64(header.Time), 0).Sub(time.Now()) // nolint: gosimple
+	delay := time.UnixMilli(int64(header.Time)).Sub(time.Now()) // nolint: gosimple
```

Comment:
Does the `wiggle` variable below, which is added to `delay`, also need changing?

Reply:
I've reduced it to a 100ms.